### PR TITLE
fix/style: correct broken links and use md

### PR DESCRIPTION
**Pull request by thephez**
There were several incorrect / malformed links in these DIPs

### CODE REVIEW

**Review by HashEngineering — DISMISSED** [review]
ACK

File: dip-0006.md [hunk 1]
```diff
@@ -361,7 +361,7 @@ The internal Dash message name is `qfcommit` and the format of the message is:
 | quorumSig | BLSSig | 96 | Recovered threshold signature
 | sig | BLSSig | 96 | Aggregated BLS signatures from all included commitments
 
-`qfcommit` with version 2 or 4 include the `quorumIndex` field highlighted below as described in [DIP24](dip-0024-rotation.md#quorum-composition):
+`qfcommit` with version 2 or 4 include the `quorumIndex` field highlighted below as described in [DIP24](dip-0024.md#quorum-composition):
```
Comment (knst) [inline dip-0006.md:364]: may be make title more detailed?

```
-`qfcommit` with version 2 or 4 include the `quorumIndex` field highlighted below as described in [DIP24](dip-0024.md#quorum-composition):
+`qfcommit` with version 2 or 4 include the `quorumIndex` field highlighted below as described in [DIP24. Quorum Composition](dip-0024.md#quorum-composition):
```
?

File: dip-0008.md [hunk 2]
```diff
@@ -341,10 +341,10 @@ computed by counting the number of outcomes where the attacker controls
 160 or more nodes in the quorum divided by the total number of outcomes.
 This is computed by the sum
 
-<p align="center"><img src="dip-0008/fin_sum.gif" alt="Final sum" ></p>
+![Final sum](dip-0008/fin_sum.gif)
 
 The numbers involved in this sum can be quite large. We may
-use a <A HREF="dip-0008/quorum_attack.py">Python Script</A> to effectively
```
Comment (knst) [inline dip-0008.md:347]: also maybe Python Script quorum_attack.py ?

**Review by knst — APPROVED** [review]
I proposed 2 suggestions, but they are not blocker for fixes.

testedACK

File: dip-0008.md [hunk 3]
```diff
@@ -341,10 +341,10 @@ computed by counting the number of outcomes where the attacker controls
...
+use a [Python Script](dip-0008/quorum_attack.py) to effectively
```
Comment (thephez) [inline dip-0008.md:347]:
```suggestion
use the [quorum_attack.py script](dip-0008/quorum_attack.py) to effectively
```

**Comment by knst** [comment]
ACK

File: dip-0006.md (hunk 1 above)
Comment (thephez) [inline dip-0006.md:364]:
```suggestion
`qfcommit` with version 2 or 4 include the `quorumIndex` field highlighted below as described in the [DIP24 Quorum Composition section](dip-0024.md#quorum-composition):
```

**Review by knst — APPROVED** [review]
ACK

**Review by UdjinM6 — APPROVED** [review]

**Pull request merged**